tests/integration/sugar/test_network_id.py

```diff
@@ -14,7 +14,7 @@ class TestNetworkID(TestCase):
     # Autofill should override tx networkID for network with ID > 1024
     # and build_version from 1.11.0 or later.
     def test_networkid_override(self):
-        with WebsocketClient("wss://hooks-testnet-v3.xrpl-labs.com") as client:
+        with WebsocketClient("wss://s.altnet.rippletest.net:51233") as client:
             wallet = generate_faucet_wallet(client, debug=True)
             tx = AccountSet(
                 account=wallet.classic_address,
```

Comment:
You can use `sidechain-net2.devnet.rippletest.net` and `rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh`